bots/pr/src/main/java/org/openjdk/skara/bots/pr/ContributorCommand.java

```diff
@@ -87,7 +87,7 @@ private Optional<EmailAddress> parseUser(String user, PullRequest pr, CensusInst
         }
 
         if (contributor.fullName().isPresent()) {
-            return Optional.of(EmailAddress.from(contributor.fullName().get(), contributor.username() + "@openjdk.org"));
+            return Optional.of(EmailAddress.from(contributor.fullName().get(), contributor.username() + "@" + censusInstance.configuration().census().domain()));
```

Comment:
Could do with a line break here to make it more readable, especially in diffs.

Reply:
Sure